### ISSUE TITLE
LUKA  daily basic income dapp

### ISSUE THREAD

**Issue by dumpa** [Blockternship/projects#12]
Hi all :)
I'm dumpa, social entreprenuer, thinker, writer. I have thought a lot about how ethereum can be used for social good and have a couple of ideas. 

**LUKA**
It is a dapp that will guarantee that everyday you have luka to satisfy your basic needs.

"You cannot start the day with less than 20 (or whatever minLuka) LUKA" 
If you spend your luka, the next morning you will have 20. Luka will be re-distributed every day so that no one has less than 20. You take from the extreme rich and give some to the poor. Everybody opens a wallet and gets 100 luka. 

**Why**
If everybody has a basic income they can say no to things/jobs/activities that keep them in poverty. With a basic income they can say no to things they have but don't want. They can work in whatever they want, and don't get distracted doing things because they need food, shelter, medicine...

We can create a currency that can help people satisfy basic needs. It won't be used for everything just the basic.

**Economic incentives**

- You want money to move around. luka doesn't encourage you holding your money but spending it.
If you have little you better use it so it topples up tomorrow. If you have a lot you will use it so that it is not taken away to be re-distributed. People will use it fast.
- Many people live by the day, so I like the idea of people having a daily and not monthly salary. It is more realistic to how poor people think and do.

**Your area of expertise or what you're interested in learning**
I want to re-learn to code. Have done some coding going through my BSc, MSc, and Phd. And 20 years later the world changed. 
I used to study with complexity, game theory, simulations social behaviour, and later became an entrepreneur with a 3d printing open source company, and a design lab focused on social innovation. I have worked with rural, indigenous, and marginal communities in Colombia. 
Would love to see a dapp like this one on the streets.

**What you think you'll need**
I need help with coding, will put all the effort but i'm rusted. Any help welcome.
Also code audits, it's a smart contract giving away money.
Need people with whom to discuss the economics, I have the idea and in my mind it works, but still don't know how the parameter details will affect the dynamics of the economy.
Will need also to think about how to market or turn into a pilot program.

**Contact info** 

https://github.com/dumpa
https://dumpa.co
https://twitter.com/dumpita
juanpablocalderon at gmail

**Comment by YalorMewn** [comment]
Hey @dumpa I would like to invite you to the first call of the hackathon, would you send me a quick email here YalorTackson@gmail.com also you can add yourself to the event [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=NjlxM2dtdDd2bHA5ODNxbzNsYm5vOTMyZDkgeWFsb3J0YWNrc29uQG0&tmsrc=yalortackson%40gmail.com)

**Comment by deamme** [comment]
@dumpa Seems like you got two proposals (this and the other one). Is there a specific one you'd rather want to work on?

**Comment by dumpa** [comment]
@deamme 

tl;dr do I have to choose one? when would the deadline to choose be? I'm trying to pick one if I definitely have too (or feel I better have just one), just need a bit to evaluate pros and cons. 

I'm having some trouble answering this question. I will, don't worry. 
I love both projects. And my purpose on Blockternship is to learn to code while creating a powerful application. So both apply. 
Both are also a real challenge, and that is what makes your question relevant. 

I'm thinking about which would I rather pick and for this I'm doing a table just comparing both projects with my own variables.

I'm going to have time, and I have two different teams. Though, is true that if I work on one, and not 2, the probability of success is higher.

Do I have to choose?
Do you advice me to do so?

**Comment by YalorMewn** [comment]
Hi @dumpa we are going to do a final vote to narrow down the candidates, It is my guess that we will select one of the 2 projects that you have submitted to participate in the Blockternship, we will let you know by the 10th which one it is. That is not to discourage you from working on both, just that we only have enough resources to support one project per contributor.

**Comment by YalorMewn** [comment]
#13

**Comment by YalorMewn** [comment]
You're project has been approved @dumpa  ✨

Would you like to start a fresh repo or fork an existing one into this org ?

**Comment by dumpa** [comment]
Nice!
Let's start a new one :)

**Comment by YalorMewn** [comment]
Here you go, https://github.com/Blockternship/Luka you are Admin :)

**Comment by YalorMewn** [comment]
By the way @dumpa your mentor is @Dan-Nolan from Chain Shot. You can reach him on [Slack](https://join.slack.com/t/chainshotnodes/shared_invite/enQtMzU3ODc5NTM3MTI3LTFlZTY1YzcwM2QzYWI0ODY2ZDczMmYzOTVlYWQwZjkyZDFlYzUxZWM4NDNlNjk3N2EyNGMwOGQ0ZTVkZjQyNjE)

**Comment by Dan-Nolan** [comment]
Awesome! Excited to help in any way I can. Like Yalor mentioned feel free to reach out on Slack, tag me on Github or my keybase is @beingdannolan

**Comment by status-open-bounty** [comment]
Current balance: 0.0 ETH
Contract address: [0x6f0ea4fee71edd8b73789b0f199b5400e860cae7](https://etherscan.io/address/0x6f0ea4fee71edd8b73789b0f199b5400e860cae7)
![QR Code](https://openbounty.status.im/qr/Blockternship/projects/bounty/12/c1cf0f933403c839ed579cdeb53908e23fc5f28e0df57757444a844d26d97588/qr.png)
Network: Mainnet
To claim this bounty sign up at [https://openbounty.status.im](https://openbounty.status.im) and make sure to update your Ethereum address in `My Payment Details` so that the bounty is correctly allocated.
To fund it, send ETH or ERC20/ERC223 tokens to the contract address.
